src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/UserRolesControllerTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableList;
+import uk.gov.hmcts.cmc.claimstore.exceptions.NotFoundException;
+import uk.gov.hmcts.cmc.claimstore.services.UserRolesService;
+import uk.gov.hmcts.cmc.domain.models.UserRole;
+import uk.gov.hmcts.cmc.domain.models.UserRoleRequest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UserRolesControllerTest {
```

Comment:
I find this test useless and agains the convention of testing endpoints via HTTP calls. Please remove.